src/plugins/typescript.ts

```diff
@@ -32,15 +32,31 @@ export interface ITypescriptPluginData {
 }
 
 export class TypescriptPlugin implements IPlugin<ITypescriptPluginData> {
+    public constructor(
+        /**
+         * Options to `ts-quick-docs`, mostly for customizing which symbols appear in the output.
+         */
+        private options: IDocumentationOptions = {},
+
+        /**
+         * Compiler options for Typescript program used to "read" your typings.
+         * (This is distinct from whatever options you need to build your typings.)
+         * If omitted, the default compiler options are used.
+         */
+        // HACK: using any to avoid duplicate typings issue with ts.CompilerOptions
```

Comment:
i think a mismatch in versions between `typescript` used here and `typescript` used in ts-quick-docs, but i didn't look closely. it was one of those "definitions of X and X are not the same" (but they _are_).